DOCKER/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+if [[ $BARAK_SEED ]]; then
+  cat /go/src/github.com/eris-ltd/eris-db/Godeps/_workspace/src/github.com/tendermint/tendermint/cmd/barak/$BARAK_SEED | barak &
```

Comment:
duh. ye, mint is there in the container, but i'm reworking this now and adding mint-client so will update....